frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu_user.dart

```diff
@@ -84,15 +84,61 @@ class MenuUser extends StatelessWidget {
   }
 
   Color _generateRandomNameColor(String name) {
-    int hashCode = name.hashCode;
-    Random random = Random(hashCode);
-    Color color = Color(random.nextInt(0xffffffff));
-    double luminance = color.computeLuminance();
-    // Checking if the color is too light and returning the darker version
-    if (luminance > 0.5) {
-      return color.withOpacity(0.8);
+    final hash = name.hashCode;
+    final h = _normalizeHash(
+      hash,
+      0,
+      360,
+    );
+    final s = _normalizeHash(
+      hash,
+      50,
+      75,
+    );
+    final l = _normalizeHash(
+      hash,
+      25,
+      60,
+    );
+    final color = _hslToColor(
+      [h, s, l],
+    );
+    return color;
+  }
```

Comment:
I think we can make this more readable by removing commas for formatting.

Reply:
@Xazin Removed the trailing commas, should be fine now